x-pack/plugins/fleet/public/components/unverified_package_modal.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiConfirmModal, EuiCallOut } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+import type { PackageInfo } from '../../common';
+
+export const UnverifiedPackageModal: React.FC<{
+  onCancel: () => void;
+  onConfirm: () => void;
+  pkg?: Pick<PackageInfo, 'name' | 'version'>;
+}> = ({ onCancel, onConfirm, pkg }) => {
+  const title =
+    pkg && pkg.name && pkg.version
+      ? i18n.translate('xpack.fleet.unverifiedPackageModal.calloutTitleWithPkg', {
+          defaultMessage: 'Integration {pkgName}-{pkgVersion} has failed verification',
+          values: {
+            pkgName: pkg.name,
+            pkgVersion: pkg.version,
+          },
+        })
+      : i18n.translate('xpack.fleet.unverifiedPackageModal.calloutTitleNoPkg', {
+          defaultMessage: 'The integration has failed verification',
+        });
+  // TODO: add link to docs
+  return (
+    <EuiConfirmModal
+      title={
+        <span className="eui-textBreakWord">
+          <FormattedMessage
+            id="xpack.fleet.unverifiedPackageModal.title"
+            defaultMessage=" Install unverified integration?"
```

Comment:
Not sure if the extra space here is problematic or not.

```suggestion
            defaultMessage="Install unverified integration?"
```